frame/beefy-mmr/src/lib.rs

```diff
@@ -72,19 +72,9 @@ where
 pub struct BeefyEcdsaToEthereum;
 impl Convert<beefy_primitives::crypto::AuthorityId, Vec<u8>> for BeefyEcdsaToEthereum {
 	fn convert(a: beefy_primitives::crypto::AuthorityId) -> Vec<u8> {
-		use k256::{elliptic_curve::sec1::ToEncodedPoint, PublicKey};
```

Comment:
I think this was the only user of `k256` within `beefy-mmr`. You should update the manifest.